pyproject.toml

```diff
@@ -1,3 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "nav-argus-glue"
+authors = [{name = "Morten Brekkevold", email = "morten.brekkevold@sikt.no"}]
+description = "An Argus glue service for Network Administration Visualized"
+keywords = ["api", "argus", "client"]
+license = {text = "GPLv3"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+]
+dependencies = [
+    "simple_rest_client==1.0.8",
+    "argus-api-client>=0.4.2",
+    "pyaml",
+    "nav@git+https://github.com/Uninett/nav/tarball/5.6.0#egg=nav-5.6.0",
```

Comment:
Note in the readme if it isn't already that Nav must have been properly installed and set up *before* navargus is installed because this line will drag in Nav in a non-running fashion.

Interestingly, non-distro packaging systems like wheel/egg can't do a lot of the things Nav and other complex systems need, and I don't think there is a place for it either in language-specific systems. There's this interesting thread from the rust-world about it:
[Support installing manpages (and potentially other files)]( https://github.com/rust-lang/cargo/issues/2729).

Reply:
.. though,this gives me an idea for Nav.

Reply:
It is already noted